command/hcl2_upgrade.go

```diff
@@ -1158,6 +1158,7 @@ var PASSTHROUGHS = map[string]string{"NVME_Present": "{{ .NVME_Present }}",
 	"WinRMPassword":              "{{ .WinRMPassword }}",
 	"DefaultOrganizationID":      "{{ .DefaultOrganizationID }}",
 	"HTTPDir":                    "{{ .HTTPDir }}",
+	"HTTPContent":                "{{ .HTTPContent }}",
```

Comment:
Not sure if this one was necessary but since `HTTPDir` is there, I'm putting it

Reply:
Better be safe!